.github/workflows/ci.yaml

```diff
@@ -70,62 +64,43 @@ jobs:
       fail-fast: false
       matrix:
         images:
-        - swift:5.3
-        - swiftlang/swift:nightly-master
+        - swift:5.10
     container:
       image: ${{ matrix.images }}
     env:
       MAX_ALLOCS_ALLOWED_base64_decoding: 1000
       MAX_ALLOCS_ALLOWED_base64_encoding: 1000
+      MAX_ALLOCS_ALLOWED_base32_decoding: 1000
+      MAX_ALLOCS_ALLOWED_base32_encoding: 1000
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v4
     - name: Test
       run: ./run-tests.sh
       working-directory: ./IntegrationTests
 
   "macOS-Tests":
-    runs-on: macOS-latest
+    runs-on: macOS-13
```

Comment:
Why not `macos-14`?

Reply:
Is it out?

Reply:
Yeah, been available in GH's runners for months now

Reply:
(c.f. https://github.com/vapor/ci/blob/main/.github/workflows/run-unit-tests.yml#L162-L171)